src/GovUk.Education.ExploreEducationStatistics.Data.Processor/Services/DataArchiveService.cs

```diff
@@ -44,11 +44,11 @@ await using (var stream = dataFile.Open())
                     releaseId: releaseId,
                     stream: stream,
                     fileType: ReleaseFileTypes.Data,
-                    fileName: dataFile.Name,
+                    fileName: dataFile.Name.ToLower(),
                     contentType: "text/csv",
                     metaValues: GetDataFileMetaValues(
                         name: blob.Name,
-                        metaFileName: metadataFile.Name,
+                        metaFileName: metadataFile.Name.ToLower(),
```

Comment:
No need to do this as the `GetDataFileMetaValues` already lower cases the meta file name

Reply:
true - reverted